elevate/forms.py

```diff
@@ -18,12 +18,13 @@ class ElevateForm(forms.Form):
         label=_('Password'), widget=forms.PasswordInput(attrs={'autofocus': True})
     )
 
-    def __init__(self, user, *args, **kwargs):
+    def __init__(self, request, user, *args, **kwargs):
+        self.request = request
         self.user = user
         super().__init__(*args, **kwargs)
 
     def clean_password(self):
         username = self.user.get_username()
-        if auth.authenticate(username=username, password=self.data['password']):
+        if auth.authenticate(request=self.request, username=username, password=self.data['password']):
```

Comment:
This line is too long and that's what's causing flake8 to fail.

Reply:
Sounds good, formatted the line shorter and added the RELEASE doc. Let me know if there's anything else!